README.md

```diff
@@ -20,6 +20,13 @@ bundle exec rails server
 
 Visit http://localhost:3000 and login with email `oliver@example.com` and password `welcome`.
 
+## Docker for development environment
+
+1. Clone this repo by running `git clone https://github.com/bigbinary/wheel.git`
+2. `cd wheel` and then run `docker-compose up`.
+   Though the application would come up, we haven't created and seeded the database. Do `ctrl+c` and then run `docker-compose run app rails setup` to create and seed the database.
+3. From now onwards, we can just run `docker-compose up` from within the root of the `wheel` directory to bring up the application in a container.
+
```

Comment:
Let's mention about how to clear docker cache and get rid of weird errors. **Please do remove the `\` and `----> sentence` in the code blocks in suggestion, which i just used to to show the markdown within another markdown.**
```suggestion
# Fixing Docker Cache Issues

Sometimes, when we rebuild the container,  there are chances that the already existing cache can make the build process go awry. Thus, it's preferred to clean out the containers and rebuild from scratch in such scenarios.

Steps to re-build by removing only necessary docker data:

- Delete the containers, if they exist. Check using `docker ps -a` and then `docker rm $container_ids_to_be_removed`.
- Delete the docker images for wheel. Run `docker images -a` to see the ID's. And then cherry pick and run `docker rmi -f $image_ids`.
- Delete the volumes of wheel. Run `docker volume ls` to see the volumes. Pick the necessary ones and do `docker volume rm -f $volume_ids`
- Then remove temp files docker might've cached by `sudo rm -rf ./tmp ./log` from **root of wheel project**, in dockerize-wheel branch. The dot(`.`) is important in `./tmp` path.
- Then redo `docker-compose up --build`

If you want to try out something slightly more daring, yet effective, then run the following single line command to wipe all of the docker data, which includes containers+images+volumes and then did docker-compose up --build and things started working for me in wheel.

**Warning: The following command will wipe all of docker data of all local docker projects and containers:**

\``` -------> remove the slash (\) and this arrow and sentence
docker rm -f $(docker ps -a -q) && docker rmi -f $(docker images -q) && docker volume rm -f $(docker volume ls -q)
\``` -------> remove the slash (\) and this arrow and sentence
```